### PR TITLE
feature/camv19/katgui/cbf-health

### PR DESCRIPTION
**Pull request by lvdheever**
Minor change to show the sensor name in the config health display.

@ofaletse This is the only katgui change that I needed to get camv19 working with the new CBF config views. Can you use this branch to add your refresh/resubscribe change to camv19. Then we can take it out to site as soon as that is done.

Also made the change to use config_views instead of custom_views in katcamconfig to make it possible to use the same status_trees config in camv19 and master (as I think we will need this going forward and working on correlator views).

This needs to be merged together with related PR now - https://github.com/ska-sa/katcamconfig/pull/442

### CODE REVIEW

**Comment by ofaletse** [comment]
Okay cool.
Will commit. soon.

**Comment by ajoubertza** [comment]
Are you still going to rebuild and upload the minified files to the dist/ folder?

**Comment by lvdheever** [comment]
I was waiting for Ofaletse's changes before uploading those, but since I wanted to test this feature branch on vkaroo_mkat I had to load them. So available now, will replace once OJ has made his commits.

File: dist/kat-version.txt [hunk 1]
```diff
@@ -1 +1,5 @@
+<<<<<<< HEAD
```
Comment (ajoubertza) [inline dist/kat-version.txt]: These kind of lines "<<<<<", "=======", and ">>>>>>>" were created by git.  Result of a merge conflict.  They should not be checked in.   They indicate sections of the code where git couldn't resolve the conflict.  It shows you both options, and you must decide which to use.

**Comment by ofaletse** [comment]
@ajoubertza  please see: 063df086a68b8e1fd0c1a051565bf8d5b3f84c13

**Comment by ajoubertza** [comment]
@ofaletse OK, thanks - sorry didn't see that one.

**Comment by ofaletse** [comment]
Look ok to me.

**Pull request merged**